test/integration/snapshotting.js

```diff
@@ -15,12 +15,14 @@
 'use strict';
 
 const Promise = require('bluebird');
+const _ = require('lodash');
 const elements = require('lisk-elements').default;
 const randomUtil = require('../common/utils/random');
 const accountsFixtures = require('../fixtures/accounts');
 const queriesHelper = require('../common/integration/sql/queriesHelper.js');
 const localCommon = require('./common');
 
+global.constants = _.cloneDeep(global.constants);
```

Comment:
once you move this logic to setup then we don't need this, also you can remove the lodash require. In future, if you want to use lodash, its exposed globally for the test, you can directly use `_`.